### DISCUSSION TITLE
Playlist (mis)behaviour

### DISCUSSION THREAD

**Discussion by paul2berry** [quodlibet/quodlibet#4587] [General]
As a new user, I'm having a bit of trouble with the behaviour of created playlists.
In the Track List browser, I can create a query like this:
&(artist = “Rolling Stones”, #(rating = 1.0))
Behaves as expected and returns Rolling Stones tracks with a 5* rating (Lots)
I can then 'right click' and create a playlist.
If I then go to the Playlist browser and choose the created playlist, it returns only one track.
I have created five playlists which exhibit the same behaviour.
My feeling is that there's a 'global' switch/filter enabled in the Playlist browser.
Will somebody give me a clue?
A solution would be even better :-)

**Comment by declension** [comment]
Hmm, seems to work for me.

Did you select all before right clicking to create a playlist? Otherwise you'll just make a playlist from the selected song...

**Comment by paul2berry** [comment]
That worked.Thanks Nick 😁😁👍Sent from my Galaxy
-------- Original message --------From: Nick Boultbee ***@***.***> Date: 09/10/2024  20:53  (GMT+00:00) To: quodlibet/quodlibet ***@***.***> Cc: paul2berry ***@***.***>, Author ***@***.***> Subject: Re: [quodlibet/quodlibet] Playlist (mis)behaviour (Discussion #4587) 
Hmm, seems to work for me.
Did you select all before right clicking to create a playlist? Otherwise you'll just make a playlist from the selected song...
